go.mod

```diff
@@ -3,19 +3,17 @@ module kubesphere.io/fluentbit-operator
 go 1.13
```

Comment:
This also needs to be upgraded to `1.16`.

Reply:
Update Go version in `README.md` and `go.mod` files